docs/Assets.md

```diff
@@ -0,0 +1,105 @@
+# Assets and JSON module imports
+
+UI5 Web Components aim to be feature rich and with a minimal code footprint at the same time. In order to achieve this, 
+most UI5 Web Components packages ship their assets as `.json` files while also providing respective public module imports for them.
+
+The assets in question could be i18n texts, icons, additional themes parameters, CLDR, etc...
+
+Currently our npm packages follow the scheme:
+
+`@ui5/<PACKAGE_NAME>/dist/assets/*`
+(for the assets themselves)
+
+`@ui5/<PACKAGE_NAME>/dist/json-imports/*`
+(for the modules that provide the assets)
+
+<a name="packages"></a>
+## Packages
+
+### `base` package
+
+Asset type | Asset path | Public Module Import
+------|------------------|---
+CLDR | `@ui5/webcomponents-base/dist/assets/cldr/*` | `@ui5/webcomponents-base/dist/json-imports/LocaleData.js`
+
+Usually you don't need to import the CLDR assets from the `base` package, as the other packages provide handy shortcuts for the purpose.
+
+### `main` package
+
+Asset type | Asset path | Public Module Import
+------|------------------|---
+i18n | `@ui5/webcomponents/dist/assets/i18n/*` | `@ui5/webcomponents/dist/json-imports/i18n.js`
+Additional themes | `@ui5/webcomponents/dist/assets/themes/*` | `@ui5/webcomponents/dist/json-imports/Themes.js`
+CLDR | N/A | `@ui5/webcomponents/dist/json-imports/LocaleData.js`
+
+The `main` package does not have any additional CLDR assets besides the CLDR assets of the `base` project.
+Therefore its JSON import for CLDR is only there for convenience and simply imports the one from the `base` project.
```

Comment:
Let's remove that it is for convenience, this is actually an implementation detail that we don't want to expose and users should import the `main` assets

Reply:
done